modules/core/src/main/java/org/apache/ignite/IgniteSnapshot.java

```diff
@@ -74,6 +74,18 @@ public interface IgniteSnapshot {
      */
     public IgniteFuture<Void> createDump(String name, @Nullable Collection<String> cacheGroupNames);
 
+    /**
+     * Creates cache groups dump.
+     * Dump is consistent entry by entry backup of cache group content.
+     * In-memory cache groups also supported.
+     *
+     * @param name Dump name.
+     * @param cacheGroupNames Cache groups to include in snapshot or {@code null} to include all.
+     * @param zip If the dump file should be zipped.
```

Comment:
Let's rename this flag to `compress`

Reply:
Let's leave usage of new `compress` flag to the cli tools and don't flood out java public API.